src/main/scala/beam/agentsim/infrastructure/taz/TAZTreeMap.scala

```diff
@@ -132,6 +132,21 @@ object TAZTreeMap {
     QuadTreeBounds(minX, minY, maxX, maxY)
   }
 
+  private def quadTreeExtentFromList(lines: Seq[TAZ]): QuadTreeBounds = {
```

Comment:
can we reuse existing code?

Reply:
Created a type class beam.utils.matsim_conversion.ShapeUtils.HasQuadBounds and methods beam.utils.matsim_conversion.ShapeUtils#quadTreeBounds for finding QuadTreeBounds